rotkehlchen/api/rest.py

```diff
@@ -1555,6 +1555,68 @@ def process_history(
         result_dict = _wrap_in_result(result=result, message=msg)
         return api_response(result_dict, status_code=status_code)
 
+    def _process_history_debug(
+        self,
+        from_timestamp: Timestamp,
+        to_timestamp: Timestamp,
+        path: Optional[Path],
+    ) -> Response:
+        error_or_empty, events = self.rotkehlchen.events_historian.get_history(
```

Comment:
shouldn't you check if `error_or_empty` is not `''` and do something if it's not?